### PR TITLE
Photon commander

### PR DESCRIPTION
**Pull request by superheropug**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -22,6 +25,7 @@
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
   private RobotContainer m_robotContainer;
+  //public PhotonVisionManager photon;
```
Comment (Zandar-Double07) [inline src/main/java/frc/robot/Robot.java]: Is this comment needed?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -37,9 +43,14 @@ public class RobotContainer {
   private final Dual_Joysticks Dual_Joysticks = new Dual_Joysticks(DriveSubsystem);
   private final DriveForward DriveFiveRotations = new DriveForward(DriveSubsystem, 5.0, 0.5);
 
-  private final DriveBackward DriveThreeRotoations = new DriveBackward(DriveSubsystem, 3.0, 0.25);
-  private final TurnLeft Turn90Degrees = new TurnLeft(DriveSubsystem, navchipManager, 90, 0.5);
-  private final TurnRight Turn180Degrees = new TurnRight(DriveSubsystem, navchipManager, 180, 0.3);
+  private final DriveBackward DriveThreeRotoations = new DriveBackward(DriveSubsystem,3.0, 0.25 );
```
Comment (Zandar-Double07) [inline src/main/java/frc/robot/RobotContainer.java:46]: Keep the old turning commands, you can keep the new additions

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -85,4 +101,11 @@ public Command getAutonomousCommand() {
     // An ExampleCommand will run in autonomous
     return autoBalance;
   }
+  /**
+   * @return
+   */
+  public PhotonVisionManager getPhoton(){
```
Comment (Zandar-Double07) [inline src/main/java/frc/robot/RobotContainer.java:107]: Can you explain what this is used for?

**Review by Zandar-Double07 — CHANGES_REQUESTED** [review]
Looks mostly good, just clean up a couple of things before we merge.

**Review by Atika2022 — APPROVED** [review]
The code seems fine and it works

**Pull request closed without merging**